ui/cypress/e2e/notificationRules.test.ts

```diff
@@ -147,96 +151,144 @@ describe('NotificationRules', () => {
     })
   })
 
-  // TODO(desa): this needs to be skipped until https://github.com/influxdata/influxdb/issues/14799
-  it.skip('can create a notification rule', () => {
-    const ruleName = 'my-new-rule'
-    cy.getByTestID('alert-column--header create-rule').click()
+  for (let i = 0; i < 100; i++) {
```

Comment:
I'll remove this if these all pass

Reply:
is that what took longer than 10 mins in the e2e tests?